### PR TITLE
Fix print / article views

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| Fixed tickets | N/A
| License       | MIT

### Before 

before

print_before

### After

after

print_after

### CODE REVIEW

**Comment by tcitworld** [comment]
Can you hide the :pencil2: on the print view ?

**Comment by nicosomb** [comment]
@tcitworld oh yes you're right. Will do that now.

**Comment by nicosomb** [comment]
Done.

**Comment by tcitworld** [comment]
:+1:

**Comment by j0k3r** [comment]
What about baggy? :trollface:

**Comment by nicosomb** [comment]
ba... what? 

Oops, will fix it too.

**Comment by nicosomb** [comment]
@j0k3r Done.

**Comment by tcitworld** [comment]
For some reason Firefox doesn't wants me to print more than one page (maybe because of [this](http://www.ghacks.net/2016/06/17/firefox-49-simpify-page-printing/)). Chrome works okay.

**Comment by tcitworld** [comment]
Baggy for me. Why not one line.
wllbg_metadata

**Comment by j0k3r** [comment]
@tcitworld maybe because we need more clearer separator between each information?

**Comment by tcitworld** [comment]
Sure, but not `\n` :D

File: app/Resources/static/themes/material/css/print.css [hunk 1]
```diff
@@ -26,10 +26,23 @@
...
+  #slide-out,
+  .progress, .hide-on-large-only,
+  #article > aside,
+  #article .mbm a
+  {
     display: none !important;
   }
 
+  main {
+      padding-left: 0 !important;
```
Comment (j0k3r) [inline app/Resources/static/themes/material/css/print.css]: 2 spaces instead of 4

Comment (tcitworld) [inline app/Resources/static/themes/material/css/print.css]: I'll add a css linter for the assets validation.

**Comment by nicosomb** [comment]
@j0k3r @tcitworld I fixed indentation.

**Comment by j0k3r** [comment]
👍

**Pull request merged**